unstable/wlr-surface-layers-unstable-v1.xml

```diff
@@ -0,0 +1,231 @@
+        depths. The compositor may choose under which circumstances to send
+        inputs. By convention, the compositor will send input events to
+        the top-most surface of the top-most layer.
+
+        Interactivity is double-buffered, see wl_surface.commit.
+      </description>
+      <arg name="input_types" type="uint" enum="wl_seat::capability" summary="mask of input devices to use"/>
+      <arg name="exclusive_types" type="uint" enum="wl_seat::capability" summary="mask of input devices to use exclusively"/>
+    </request>
+
+    <request name="set_anchor">
+      <description summary="configures the anchor point of the surface">
+        Requests that the compositor anchor the surface to the specified edges
+        and corners. If two orthoginal edges are specified (e.g. 'top' and
+        'left'), then the anchor point will be the intersection of the edges
+        (e.g. the top left corner of the output); otherwise the anchor point
+        will be centered on that edge, or in the center if none is specified.
+
+        Anchor is double-buffered, see wl_surface.commit.
+      </description>
+      <arg name="anchor" type="uint" enum="anchor"/>
+    </request>
+
+    <request name="set_exclusive_zone">
+      <description summary="configures the exclusive geometry of this surface">
+        Requests that the compositor avoids occluding an area of the surface
+        with other surfaces. The compositor's use of this information is
+        implementation-dependent - do not assume that this region will not
+        actually be occluded.
+
+        This value is only meaningful if the surface is anchored to an edge,
+        rather than a corner. The zone is the number of pixels from the edge
+        that are considered exclusive.
+
+        Exclusive zone is double-buffered, see wl_surface.commit.
+      </description>
+      <arg name="zone" type="uint"/>
+    </request>
+
+    <request name="set_margin">
+      <description summary="sets a margin from the anchor point">
+        Requests that the surface be placed some distance away from the anchor
+        point on the output, in pixels.
+
+        Margin is double-buffered, see wl_surface.commit.
+      </description>
+      <arg name="horizontal" type="int"/>
+      <arg name="vertical" type="int"/>
+    </request>
+
+    <request name="get_popup">
+      <description summary="assign this layer_surface as an xdg_popup parent">
+        This assigns an xdg_shell::xdg_popup's parent to this layer_surface.
+        This popup should have been created via xdg_surface::get_popup with the
+        parent set to NULL, and this request must be invoked before committing
+        the popup's initial state.
+
+        See the documentation of xdg_popup for more details about what an
+        xdg_popup is and how it is used.
+      </description>
+      <arg name="popup" type="object" interface="xdg_shell::xdg_popup"/>
```

Comment:
Should be `xdg_popup` only, no need for that `xdg_shell` (will help the automatic documentation generator).